src/core/index.ts

```diff
@@ -1,10 +1,54 @@
+/**
```

Comment:
this part is already gone in the master: https://github.com/SebastianM/angular2-google-maps/blob/master/src/core/core-module.ts